docs/style/errors.md

```diff
@@ -0,0 +1,54 @@
+# Errors
+
+## Error messages
+
+Error messages **must begin with a lowercase letter and should not end with a period**.
+
+Examples:
+
+```rust
+// 👍 Good:
+#[derive(Debug, thiserror::Error)]
+#[error("serializing string: {0}")]
+pub struct Serialize(String);
+
+// 👎 Bad:
+// It must start with lowercase letter
+#[derive(Debug, thiserror::Error)]
+#[error("Serializing string: {0}")]
+pub struct Serialize(String);
+// It should not end with a period
+#[derive(Debug, thiserror::Error)]
+#[error("serializing string: {0}.")]
+pub struct Serialize(String);
+```
```

Comment:
Nice!

Would also make sense to include some examples using `map_error`? I know it is the same, but the code is pretty different so, maybe having both examples is useful!